Bio/PDB/PDBList.py

```diff
@@ -339,10 +342,13 @@ def retrieve_pdb_file(
         except OSError:
             print("Desired structure doesn't exists")
         else:
-            with gzip.open(filename, "rb") as gz:
-                with open(final_file, "wb") as out:
-                    out.writelines(gz)
-            os.remove(filename)
+            if compress:
+                pass
+            else:
+                with gzip.open(filename, "rb") as gz:
+                    with open(final_file, "wb") as out:
+                        out.writelines(gz)
+                os.remove(filename)
         return final_file
```

Comment:
This should probably return `filename` if `compress` is `True`.

Reply:
That's correct. It is Ok fixed now?
```suggestion
        if compress:
            return filename
```